tests/test_sentry_hook.py

```diff
@@ -134,3 +134,11 @@ def test_get_dsn_from_host_with_none(self):
         conn = Connection(conn_type="http", login=None, host="https://foo@sentry.io/123")
         dsn = get_dsn(conn)
         self.assertEqual(dsn, "https://foo@sentry.io/123")
+
+    def test_get_environment_from_extra_parameter(self):
+        """
+        Test getting environment from extra parameter
+        """
+        conn = Connection(extra='{"environment": "development"}')
+        environment = conn.extra_dejson.get("environment")
```

Comment:
Test format looks good, would also prefer a test to check if the Sentry `init()` func is being called with the correct `init_args`. You can probably just mock the `init` func out (https://docs.python.org/3/library/unittest.mock.html) and check if it was called correctly.

Reply:
Thanks for the initial review @AbhiPrasad I will update and add new test cases for `init_args` and Sentry's `init` method later this day using `unittest.mock `